source/features/user-profile-follower-badge.tsx

```diff
@@ -1,17 +1,26 @@
 import './user-profile-follower-badge.css';
 import React from 'dom-chef';
 import select from 'select-dom';
+import cache from 'webext-storage-cache';
 import * as api from '../libs/api';
 import features from '../libs/features';
 import {getUsername, getCleanPathname} from '../libs/utils';
 
-async function init(): Promise<void> {
+const userIsAFollower = cache.function(async (): Promise<boolean> => {
```

Comment:
Let's try to avoid globals, so these functions make more sense when used (`doesUserFollow(userA, userB)` instead of `doesUserFollow() /* Who?? */`)

```suggestion
const doesUserFollow = cache.function(async (userA: string, userB: string): Promise<boolean> => {
```